### PR TITLE
fix: pypi direct dependency failure

### PR DESCRIPTION
**Pull request by KrishPatel13**
**What kind of change does this PR introduce?**
Fixes: https://github.com/OpenAdaptAI/OpenAdapt/issues/457

**Summary**

**Checklist**
* [x] My code follows the style guidelines of OpenAdapt
* [x] I have performed a self-review of my code
* [x] If applicable, I have added tests to prove my fix is functional/effective
* [x] I have linted my code locally prior to submission
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation (e.g. README.md, requirements.txt)
* [x] New and existing unit tests pass locally with my changes

**How can your code be run and tested?**

**Other information**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ pip3 install poetry
 poetry install
 poetry shell
 alembic upgrade head
+python -m spacy download en_core_web_trf
```
Comment (KrishPatel13) [inline README.md:86]: Add the installation command in the Manual Setup section

File: install/install_openadapt.ps1 [hunk 2]
```diff
@@ -283,6 +283,7 @@ Set-Location .\OpenAdapt
 RunAndCheck "pip install poetry" "Run ``pip install poetry``"
 RunAndCheck "poetry install" "Run ``poetry install``"
 RunAndCheck "poetry run alembic upgrade head" "Run ``alembic upgrade head``" -SkipCleanup:$true
+RunAndCheck "poetry run python -m spacy download en_core_web_trf" "Run ``python -m spacy download en_core_web_trf``" -SkipCleanup:$true
```
Comment (KrishPatel13) [inline install/install_openadapt.ps1:286]: Add the installation command in the installation scrips

File: pyproject.toml [hunk 3]
```diff
@@ -48,7 +48,6 @@ openai = "0.27.5"
 pandas = "2.0.0"
 presidio_analyzer = "2.2.32"
 presidio_anonymizer = "2.2.32"
-presidio_image_redactor = "0.0.46"
```
Comment (KrishPatel13) [inline pyproject.toml:51]: Resolved a dependency conflict related to spacy `en_core_web_trf model`

**Review by KrishPatel13 — COMMENTED** [review]
@abrichr and @Mustaballer 

Ready for merging! PLease take a look

File: pyproject.toml (hunk 3 above)
Comment (KrishPatel13) [reply pyproject.toml:51]: Had to change `presidio-image-redactor` from 0.0.46 to 0.0.48

**Review by Mustaballer — APPROVED** [review]

**Pull request merged**